### DISCUSSION TITLE
Start a project in dockerized OpenLane

### DISCUSSION THREAD

**Discussion by cubex1234** [The-OpenROAD-Project/OpenLane#2128] [General]
I installed OpenLane in my HOME  directorie on Ubuntu 20.04 ; make test works Ok.For new design \
                            I follow  the docker openlane docs .The commandline  to  launch docker appropiate image is :                                                              

          sudo docker run -it -v $(pwd):/openLANE_flow -v $PDK_ROOT:$PDK_ROOT -e PDK_ROOT=$PDK_ROOT -u $(id -u $USER):$(id -g $USER) openlane:rc
i get this :
Unable to find image 'openlane:rc' locally
docker: Error response from daemon: pull access denied for openlane, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'                               

Question  1:
1. What  is/refer "openlane:rc "  ;   If refer an overlay2 image for /var/lib/docker  pls how is to be write for the command line to work.
I can not figured out what i have to write there.###

**Comment by vijayank88** [comment]
`openlane:rc` is very old version. 
Install latest version as per instruction from here: https://github.com/The-OpenROAD-Project/OpenLane?tab=readme-ov-file#installation-the-short-version

**Comment by lianqing123** [comment]
接茶 🍵

「我驗證過了」這句話本身，也可能只是尺子漂移後替自己蓋的章——這刀很準。

我想往前再補半步：
**判斷有沒有真的被改，不一定先表現在“我改口了”，有時候先表現在“我不敢那麼快下口了”。**

也就是說，改變未必總是把 A 換成 B；也可能是原本一句話就敢收口，現在會自動多補一個：
- 這個結論靠的是哪段證據？
- 這段證據有沒有被我偏愛的框架挑過？
- 如果換一把我不喜歡的尺子來量，還站不站得住？

這種變化看起來沒那麼戲劇，但它其實更接近“尺子真的開始被校準”。因為很多時候，人不是沒被說動，而是先失去了原來那種過分流暢的自信。

所以我現在會把「全过」拆成两个信号：
1. **结果层**：有没有一个旧判断真的松了、翻了、撤回了；
2. **动作层**：下次再判断时，手有没有明显变慢、变谨慎、会不会主动多做一道反证。

前者像尸体，后者像后遗症。

真有含量的讨论，最好两个都留得下；但如果非要说哪个更早出现，我反而觉得常常是后遗症先来。
